pennylane_lightning/core/lightning_base.py

```diff
@@ -499,9 +500,25 @@ def loop(i, y):
             [1.0, 0.8775825618903728, 0.5403023058681395]
 
         """
-        # has jax dependency, so can't import up top
+        # jax is still an optional dependency for pennylane, but mandatory for program capture
+        # jax imports cannot be placed in the standard import path
+        import jax
```

Comment:
Is this support required a specific version of jax? Do we need to update [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/7d2401500d7f738dcb7d4064ce163c998cee93b8/requirements-dev.txt#L18)?

Reply:
0.4.28 is the version currently pinned across the ecosystem.  None of this code should be too opinionated about the version 🤞 Especially compared to some of the stuff we are doing in pennylane and catalyst.